content/learn/07.built-in-libraries/02.i2s/i2s.md

```diff
@@ -113,31 +115,44 @@ The next sample of incoming I2S data available (or 0 if no data is available)
 ### `write()`
 
 #### Description
-Writes binary data to the I2S interface. This data is sent as a sample or series of samples.
+Writes binary data to the I2S interface. Data is sent either as one sample or a sequence of samples.
+
+The single argument function `write(val)` writes one sample for one channel. A full write requires
+calling this function twice, once for the left channel and once for the right channel.
+If the device buffer is full, the function blocks until the data can be written.
+Before writing, the value `val` will be converted to the appropriate number of bits per sample.
```

Comment:
It should be `n` times depending on the amount of channels, if you are only working with one channel then its only once.